wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -33,7 +33,8 @@
 _logger = logging.getLogger(__name__)
 LOG_PREFIX = f"{click.style('sched:', fg='cyan')} "
 
-DEFAULT_POLLING_SLEEP = 5.0
+DEFAULT_POLLING_SLEEP = 5.0  # 5 seconds
+PREEMPTED_POLLING_TIMEOUT = 60 * 10.0  # 10 minutes
```

Comment:
is this a crazy long amount of time?

Reply:
I think it's reasonable. EKS can take a few minutes to spin a node back up after dropping it, 10 minutes is a safe wait period